docs/network-flow-visibility.md

```diff
@@ -112,6 +61,11 @@ ClickHouse as the data storage, and use Grafana as the data visualization and mo
 
 ### Deployment Steps
 
+This section talks about detailed steps to deploy the Grafana Flow Collector
+using Helm charts or YAML manifests. For quick steps of installing Theia
+including the Grafana Flow Collector, please refer to the [Getting Started](getting-started.md)
```

Comment:
`Getting Started` link needs to be updated in `networkpolicy-recommendation.md` L29.

Reply:
Updated it.